### PR TITLE
Added Bugs section to the guidelines.

### PR DESCRIPTION
**Pull request by apicella1**
Added a bugs rule to the guidelines. It needs some fixing as there's 100% some formatting things I have no idea how to do. I apologize for putting this on the devs. 

I also suggest to put the Mojira website link where it says "Mojira website". This allows users who don't know what the Mojira website is, to view what the mojira website is so they can understand it better. 
(bugs.mojang.com/issues/)

### CODE REVIEW

**Comment by apicella1** [comment]
Shit, I didn't mean to submit this yet.

**Comment by Juknum** [comment]
You can edit your post btw

File: pages/textures/texturing-guidelines.md [hunk 1]
```diff
@@ -250,6 +251,7 @@ ___
   <li id="identical-when-upscaled"><b>If a texture would end up looking identical to vanilla when upscaled (while following all the above rules), it shall not be included in Compliance.</b> Self-explanatory. Textures like these look exactly the same as vanilla when used in-game, and so they only bloat the pack and needlessly increase the file size.</li>
   <p class="red-text" style="text-align: center;"><b>Special rules and/or exceptions can apply to specific textures if the Texture Supervision Council approves it.</b></p>
 </ol>
+<li id="bugs-in-textures"><b>If a texture has issues noted as a bug on the Mojira website, Mojang's offical Bug tracker, then changes should be made to fix them. Some examples of bugs that are allowed to be fixed include, unused/misaligned pixels or textures, Jappafication errors, or parity issues. Fixing bugs allows for the pack to have issues in textures resolved before they are added into the base game which saves us time.
```
Comment (Juknum) [inline pages/textures/texturing-guidelines.md]: `</b>` & `</li>` are missing !!

**Review by Juknum — CHANGES_REQUESTED** [review]

File: pages/textures/texturing-guidelines.md (hunk 1 above)
Comment (apicella1) [reply pages/textures/texturing-guidelines.md]: Hopefully I fixed it in this recent commit.

**Review by RobertRR11 — APPROVED** [review]

**Pull request merged**